ttnn/cpp/ttnn/operations/eltwise/unary/device/unary_composite_op.hpp

```diff
@@ -64,29 +69,32 @@ Tensor _std_overload(const Tensor&, const std::optional<MemoryConfig>&);
 Tensor _normalize(const Tensor&, const std::optional<MemoryConfig>&);
 Tensor _deg2rad(const Tensor&, const std::optional<MemoryConfig>&);
 Tensor _rad2deg(const Tensor&, const std::optional<MemoryConfig>&);
-Tensor _hardswish(const Tensor&, float, float, const std::optional<MemoryConfig>& );
-Tensor _hardsigmoid(const Tensor&, float, float, const std::optional<MemoryConfig>& );
-Tensor _hardtanh(const Tensor&, float, float, const std::optional<MemoryConfig>& );
+Tensor _hardswish(const Tensor&, float scale =  1.0f/6.0f, float shift = 0.5f, const std::optional<MemoryConfig>& output_mem_config = std::nullopt);
+Tensor _hardsigmoid(const Tensor&, float scale =  1.0f/6.0f, float shift = 0.5f, const std::optional<MemoryConfig>& output_mem_config = std::nullopt);
+Tensor _hardtanh(const Tensor&, float min = -1, float max = 1, const std::optional<MemoryConfig>& output_mem_config = std::nullopt);
```

Comment:
is this to align with torch?

Reply:
- hard swish torch reference
        -https://pytorch.org/docs/stable/generated/torch.nn.Hardswish.html

- hardsigmoid torch reference
        - https://pytorch.org/docs/stable/generated/torch.nn.Hardsigmoid.html#torch.nn.Hardsigmoid

- hardtanh reference
        - https://pytorch.org/docs/stable/generated/torch.nn.Hardtanh.html